app/models/concerns/credentials_syncable_concern.rb

```diff
@@ -6,13 +6,13 @@ module CredentialsSyncableConcern
   end
 
   def sync_credentials
-    if email_changed? || encrypted_password_changed?
+    if saved_change_to_email? || saved_change_to_encrypted_password?
```

Comment:
J'ai surement loupé un truc, mais c'est quoi le rapport de tout ces renommages avec le bump ?

Reply:
Des changements de comportement sur ces méthodes
- https://www.levups.com/en/blog/2017/undocumented-dirty-attributes-activerecord-changes-rails51.html
- http://blog.toshima.ru/2017/04/06/saved-change-to-attribute.html